packages/utils/src/object.ts

```diff
@@ -207,18 +208,33 @@ export function extractExceptionKeysForMessage(exception: Record<string, unknown
  * Works recursively on objects and arrays.
  */
 export function dropUndefinedKeys<T>(val: T): T {
```

Comment:
Small nit: Should we update the JSDoc and mention that the function handles circular references?

Reply:
Yup, thanks. Added comment describing what the function does with circ refs - whis is also implying that it handles circ refs ^^ -> 4429b6433acad1481dbe0b8573cb6e4522ba2f0c